Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -572,6 +572,15 @@ const Info<bool> MAIN_EMULATE_SKYLANDER_PORTAL{
 const Info<bool> MAIN_EMULATE_INFINITY_BASE{
     {System::Main, "EmulatedUSBDevices", "EmulateInfinityBase"}, false};
 
+const Info<bool> MAIN_EMULATE_WII_SPEAK{{System::Main, "EmulatedUSBDevices", "EmulateWiiSpeak"},
+                                        false};
+
+const Info<std::string> MAIN_WII_SPEAK_MICROPHONE{
+    {System::Main, "EmulatedUSBDevices", "WiiSpeakMicrophone"}, ""};
+
+const Info<bool> MAIN_WII_SPEAK_CONNECTED{{System::Main, "EmulatedUSBDevices", "WiiSpeakConnected"},
+                                          false};
```

Comment:
I think the source code name of this setting is confusing, since the obvious interpretation is that it controls whether the Wii Speak is connected to the console, which isn't actually the case, as it controls whether a *host* microphone is connected. My suggestion for a clearer name would be `MAIN_WII_SPEAK_MUTED` (with an inverted meaning), but I'm open to other options too (maybe something that makes it more obviously connected to `MAIN_WII_SPEAK_MICROPHONE`?).